docs/features/react-native.mdx

```diff
@@ -9,7 +9,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 The React Native and Developer tooling teams at Facebook work in close collaboration to make sure Flipper offers top-notch value out of the box for React Native development.
 Integration between React Native and Flipper is enabled out of the box in React Native version 0.62 and higher.
 
-→ [See setup instructions for React Native](../getting-started/index#setup-your-react-native-app)
+→ [See setup instructions for React Native](https://fbflipper.com/docs/getting-started/react-native)
```

Comment:
Thanks for the fix, but could you please stick to relative URLs, i.e. `/docs/getting-started/react-native`?